npm failing to install @design-systems

**Describe the bug**

Hello!

I'm following the [instructions for installing the cli](https://intuit.github.io/design-systems-cli/#/getting-started), and am receiving an npm error when initializing `package.json`

**To Reproduce**

From an empty directory, run `npm init @design-systems`

```
> npm --version
7.5.6
> npm init @design-systems
Need to install the following packages:
  @design-systems/create
Ok to proceed? (y) y
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "link:": link:../cli-utils
```

Some research suggests that using `file` instead of `link` for local dependencies may resolve this issue. (ref: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#dependencies)

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -160,9 +160,9 @@ workflows:
           requires:
             - test
             - lint
-      - integration-ds:
-          requires:
-            - release
+      # - integration-ds:
+      #     requires:
+      #       - release
           filters:
             branches:
               ignore:

--- a/packages/create/package.json
+++ b/packages/create/package.json
@@ -26,8 +26,8 @@
     "url": "https://github.com/intuit/design-systems-cli.git"
   },
   "peerDependencies": {
-    "@design-systems/cli-utils": "link:../cli-utils",
-    "@design-systems/plugin": "link:../plugin",
+    "@design-systems/cli-utils": ">= 2.x",
+    "@design-systems/plugin": ">= 2.x",
     "change-case": ">=3.x",
     "cli-spinners": ">=2.x",
     "colorette": ">=1.x",

